packages/big-design/src/components/Input/spec.tsx

```diff
@@ -224,27 +234,58 @@ test('error shows with valid string', () => {
       <Input error="" />
     </FormGroup>,
   );
+  const styledInputWrapper = container.querySelector('[class*="StyledInputWrapper"]');
 
   expect(container.querySelector('[class*="StyledError"]')).not.toBeInTheDocument();
+  expect(styledInputWrapper).toHaveStyleRule('border', basicBorderStyle);
 
   rerender(
     <FormGroup>
-      <Input error={error} />
+      <Input label="Test" error={error} />
     </FormGroup>,
   );
 
   expect(container.querySelector('[class*="StyledError"]')).toBeInTheDocument();
+  expect(styledInputWrapper).toHaveStyleRule('border', errorBorderStyle);
 });
 
-test('error shows when an array of strings', () => {
-  const errors = ['Error 0', 'Error 1'];
-  const { getByText } = render(
-    <FormGroup>
-      <Input error={errors} />
-    </FormGroup>,
-  );
+describe('error shows when an array of strings', () => {
+  test('empty array', () => {
+    const { container } = render(
+      <FormGroup>
+        <Input error={[]} />
+      </FormGroup>,
+    );
+    const styledInputWrapper = container.querySelector('[class*="StyledInputWrapper"]');
 
-  errors.forEach((error) => expect(getByText(error)).toBeInTheDocument());
+    expect(container.querySelector('[class*="StyledError"]')).not.toBeInTheDocument();
+    expect(styledInputWrapper).toHaveStyleRule('border', basicBorderStyle);
```

Comment:
I don't think we need to test if there if the wrapper has the stylings 🤔  Querying to see if the `StyledError` container is in the document should be sufficient.

Reply:
Without this fix we have cases when the input has an error state (red border etc.) but, the `StyledError` is not in the document. So, this PR was created to fix these cases.

Reply:
Ah yeah, good callout!